app/pages/lab/workflow.cjsx

```diff
@@ -331,6 +333,57 @@ EditWorkflowPage = React.createClass
           </p>
 
           <hr />
+          
+          <div>
+            <span className="form-label">How should the project completeness be calculated?</span>
+            <br />
+            <p className="form-help">
+              <small>
+                Use this option to change how your project's completeness is caclucated on the public statistics page.
+              </small>
+            </p>
+            <p className="form-help">
+              <small>
+              When using "Classification Count" the completeness will increase after each classification. The
+              {' '}total number of classifications needed to complete the workflow is estimated assuming a constant retirement limit.
+              {' '}If the retirement limit is changed and/or subjects sets are unlinked from an <b>active</b> workflow this estimate will be inaccurate.
+              {' '}To avoid these issues completed subject sets should not be removed, instead completed workflows should be deactivated
+              {' '}and new ones created for new subject sets (note: workflows can be copied using the <i className="fa fa-copy"/> button at the top of the page).
+              </small>
+            </p>
+            <p className="form-help">
+              <small>
+              When using "Retirement Count" the completeness will increase after each image retires.  Since the images are
```

Comment:
true but we only calculate the completeness metric every hour...the UI could check it's accurate and recalculate if it's not.